README.markdown

```diff
@@ -1,4 +1,7 @@
-JSZip
+JSZip [![Build Status](https://api.travis-ci.org/Stuk/jszip.svg?branch=master)](http://travis-ci.org/Stuk/jszip) [![Code Climate](https://codeclimate.com/github/Stuk/jszip/badges/gpa.svg)](https://codeclimate.com/github/Stuk/jszip)
```

Comment:
Putting the images here break the title. Any reason to move the build status ?
Also, I wouldn't put the codeclimate badge: the final [score](https://docs.codeclimate.com/docs/gpa) makes little sense as it currently includes [generated files](https://codeclimate.com/github/Stuk/jszip/dist/jszip.min.js).